packages/dashboard/src/app/views/exploreTemplates/modal/templateDetails/karma/templateDetails.karma.js

```diff
@@ -105,6 +105,12 @@ describe('See template details modal', () => {
   }
 
   describe('Action: Navigate template details modal', () => {
+    it('should pass accessibility tests', async () => {
+      const templateDetails = fixture.screen.queryByRole('dialog');
+      expect(templateDetails).toBeTruthy();
+      await expectAsync(templateDetails).toHaveNoViolations();
```

Comment:
Just curious, why the `expectAsync`?

Reply:
yeah! so basically just so the expect knows it needs to wait on checking for violations. It's set up like this in the docs: https://github.com/nickcolley/jest-axe and it'll fail without it because the check isn't done yet.

Reply:
Awesome Thanks!